site/cypress/e2e/docker_ui.spec.js

```diff
@@ -63,7 +63,7 @@ describe('Docker UI Test', () => {
         }, 10000);
 
         // Updated time should not be "Unknown"
-        cy.get(':nth-child(1) > p')
+        cy.get('div.content').find('p').eq(1)
```

Comment:
This whole test should be slightly refactored to use better selectors, since if any UI is changed later, then this test will fail,  but changing this little selector to get the tests to pass is fine. I will create an issue to refactor the test to use better selectors.